python/GafferSceneUI/MotionPathUI.py

```diff
@@ -127,15 +130,54 @@
 
 		],
 
-		"increment" : [
+		"samplingMode" : [
+
+			"description",
+			"""
+			Use "Fixed" mode for a curve with a constant vertex count.
+
+			Use "Variable" mode for a curve sampled at regular `step` intervals.
+
+			> Note : This curve may have a changing vertex count over a frame range.
+
+			> Caution : In "Variable" mode it may not be possible to render with
+			deformation blur enabled. Be sure to disable it via `StandardAttributes`
+			if you want to render a variable sampled curve. 
+			""",
+
+			"preset:Variable Samples (steps)", GafferScene.MotionPath.SamplingMode.Variable,
+			"preset:Fixed Samples", GafferScene.MotionPath.SamplingMode.Fixed,
```

Comment:
Can we call these simply "Variable" and "Fixed"? The repetition in "Sampling Mode : Variable Samples" reads a bit oddly, at least to me. I think the activators are sufficient to mean we don't need the "(steps)" clarification either.

Reply:
I’m not particularly happy with the wording either. I’d started with “Fixed” and “Variable” but there was some confusion when I demoed it for a couple people... I think the suggestion was to change “Variable” to “Stepped” but I went with the long form to try and keep “Variable” in there...

Reply:
Addressed in 8147814a2a77e289bafad77117ec260dd912ca15